movie-app/pages/movies-list.js

```diff
@@ -0,0 +1,55 @@
+import React, { useCallback } from 'react';
```

Comment:
seems like you've forgot to integrate your next js app with styled-components: when loading page without javascript, it doesn't show any styles

Reply:
done